bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -176,6 +176,8 @@ def find_spec(self, fullname: str, path, target=None) -> ModuleSpec:
 # Register cleanup
 import atexit
 
+# Private handle to gROOT for later cleanup
+_gROOTForCleanup = cppyy.gbl.ROOT.GetROOT()
```

Comment:
Can't we initialize the variable directly within the `cleanup` function?

Reply:
Good idea! I will check if this is possible once the CI is green in a first pass

Reply:
I realized we can get this directly from the facade, which also has it.